targets/TARGET_NORDIC/TARGET_NRF5/TARGET_MCU_NRF52840/PeripheralNames.h

```diff
@@ -45,8 +45,8 @@
 extern "C" {
 #endif
 
-#define STDIO_UART_TX     TX_PIN_NUMBER
```

Comment:
Mapping STDIO_UART_TX to TX_PIN_NUMBER is giving me compilation errors on armcc and gcc.

Reply:
> Mapping STDIO_UART_TX to TX_PIN_NUMBER is giving me compilation errors on armcc and gcc.

Have not seen on master and we havent done any change, so would revert this one. Or is it because of some inclusion problem? We added to objects.h one specific nordic header file to get qspi definition.

Reply:
Is this the error?
```
[Error] mbed_retarget.cpp@123,58: invalid conversion from 'int' to 'PinName' [-fpermissive]
[Error] mbed_retarget.cpp@123,58: invalid conversion from 'int' to 'PinName' [-fpermissive]
```

Should be fixed on master via PinName  cast if that is what is missing on master. The define should have that type

Reply:
I am going to fix this on master to get propagated to feature branches.

Reply:
Done, https://github.com/ARMmbed/mbed-os/pull/5618

Reply:
As it has not moved anywhere, lets keep this in, and will be cleaned I hope with the next rebase once master gets the fix